test/Common/TestUtils.cs

```diff
@@ -177,5 +178,35 @@ public static async Task<TResult> TimeoutAfter<TResult>(this Task<TResult> task,
                 throw new TimeoutException(message);
             }
         }
+
+        /// <summary>
+        /// Creates a DataReceievedEventHandler that will wait for the specified regex and then check that
+        /// the matched group matches the expected value.
+        /// </summary>
+        /// <param name="taskCompletionSource">The task completion source to signal when the value is receieved</param>
```

Comment:
typo: received

Reply:
This is seriously one of my least favorite words - my fingers just always want to spell it recieved.